Dockerfile

```diff
@@ -2,29 +2,38 @@
 FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10 as builder
 
 WORKDIR /workspace
+
+# Copy the Makefile
+COPY Makefile Makefile
+COPY Makefile.helm.mk Makefile.helm.mk
+COPY Makefile.helper.mk Makefile.helper.mk
+COPY Makefile.specialresource.mk Makefile.specialresource.mk
+
 # Copy the Go Modules manifests
 COPY go.mod go.mod
 COPY go.sum go.sum
 
-# cache deps before building and copying source so that we don't need to re-download as much
-# and so that source changes don't invalidate our downloaded layer
-# RUN go mod download
-
 # Copy the go source
 COPY main.go main.go
 COPY api/ api/
 COPY cmd/ cmd/
 COPY controllers/ controllers/
-
 COPY pkg/ pkg/
 COPY vendor/ vendor/
 
+COPY helm-plugins/ helm-plugins/
+
 # Build
 RUN CGO_ENABLED=0 GO111MODULE=on go build -mod=vendor -a -o manager main.go
+RUN ["make", "helm-plugins/cm-getter/cm-getter"]
```

Comment:
If you `go build` the plugin you can get away with copying the makefiles, which are not used except for one target.

Reply:
Indeed, that's an artifact of many iterations in this branch. I do think that we should ultimately build the manager binary by calling `make manager`, though.